src/components/form/Form.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import InputContainer from './Form.styeld';
+import useStore from './useStore/useStore';
+import Wrapper from './Wrapper.styled';
+
+export default function Form() {
+	const addNewCard = useStore(state => state.addNewCard);
+	const {
+		register,
+		handleSubmit,
+		formState: { errors },
+	} = useForm();
+
+	const onSubmit = (data, event) => {
+		addNewCard(data);
+		event.target.reset();
+	};
+
+	const initialButtonText = 'Submit';
+	const [buttonText, setButtonText] = useState(initialButtonText);
+
+	const handleClick = () => {
+		setButtonText('Added');
+
+		setTimeout(() => {
+			setButtonText(initialButtonText);
+		}, 1000);
+	};
+
+	return (
+		<Wrapper>
+			<form onSubmit={handleSubmit(onSubmit)}>
+				<InputContainer>
+					<h4>Animal Profil</h4>
+
+					<div>
+						<input
```

Comment:
In favor of accessibility it would be nice if your input fields had a corresponding label. If you don't want them to be displayed, you can still hide them via CSS.

Reply:
is changed